api/src/main/resources/workbench-api.yaml

```diff
@@ -6878,7 +6878,7 @@ definitions:
           if its enumerated type is OTHER'
       requestAccessUrl:
         type: string
-        description: Request access url for users to use during registration.
+        description: Optional url for institution's handling their own access requests
```

Comment:
sometimes apostrophes here can cause problems.  did you notice anything running locally?

Reply:
(I did not)

Reply:
Good point. I do not recall seeing any issues. Agreed though, apostrophes can be tricky.